apps/prairielearn/assets/scripts/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "lib": ["ES2020", "DOM"],
+    "noEmit": true,
+    "rootDir": "./",
+    "outDir": "../../public/build/scripts",
```

Comment:
The only reason I had to defined an `outDir` here is this: https://github.com/body-builder/typescript-cp/issues/14